runelite-client/src/main/java/net/runelite/client/plugins/customcursor/CustomCursorPlugin.java

```diff
@@ -81,6 +94,12 @@ public void onConfigChanged(ConfigChanged event)
 		}
 	}
 
+	@Subscribe
+	public void onItemContainerChanged(ItemContainerChanged event)
+	{
+		updateCursor();
```

Comment:
Maybe do this only if the cursor is set to equipped cursor? Otherwise you are constantly reloading for example CUSTOM_IMAGE_FILE cursor etc.